manifests/upstream/keptn/helm/generated_templates/keptn/charts/control-plane/charts/mongodb/templates/deployment.yaml

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Are these not dupes ?

Reply:
This was generated from the `helm template` command. We don't use this but since these are generated, I'm leaning toward not modifying generated files. But on the other hand, this will make karina binary bigger for nothing. I'm considering moving upstream out of the manifest as well so we won't copy duplicate manifests to the binary.

Reply:
And if we just add them to a .gitignore ?

Reply:
You mean just ignore this mongodb templates `manifest/upstream/keptn/helm/generated_templates/keptn/charts/control-plane/charts/mongodb` or something else?

Reply:
Ignore anything that is not being included by kustomize

Reply:
Ok, I can do that. Will the packing step ignore the file in .gitignore too?